File: web/regos-web/src/features/regulatory/applications/types/RegulatoryApplicationDetail.ts
export interface RegulatoryApplicationDetail {
  id: string;
  name: string;
  applicationNumber: string | null;
  status: string;
  countryName: string;
  countryCode: string;
  authorityName: string;
  authorityCode: string;
  organizationName: string;
}

File: src/RegulatoryApplication/RegOS.RegulatoryApplication.Application/Queries/GetRegulatoryApplication/RegulatoryApplicationDetail.cs
namespace RegOS.RegulatoryApplication.Application.Queries.GetRegulatoryApplication;

public sealed record RegulatoryApplicationDetail(
    Guid Id,
    string Name,
    string? ApplicationNumber,
    string Status,
    string CountryName,
    string CountryCode,
    string AuthorityName,
    string AuthorityCode,
    string OrganizationName);
